test/unit/Site.test.js

```diff
@@ -36,7 +37,7 @@ test('Site Generate builds the correct amount of assets', async () => {
   const site = new Site('inner/', 'inner/_site');
   await site.generate();
   const paths = Object.keys(fs.vol.toJSON());
-  const originalNumFiles = 8;
+  const originalNumFiles = 9;
```

Comment:
@acjh Is this supposed to match the number of entries in `json`? If so, maybe we can use `Object.keys(json).length` instead?

Reply:
Indeed.